syncall/tw_caldav_utils.py

```diff
@@ -64,9 +64,20 @@ def convert_tw_to_caldav(tw_item: Item) -> Item:
         caldav_item["start"] = tw_item["due"] - timedelta(hours=1)
         caldav_item["due"] = tw_item["due"]
 
+    # Tags
     if "tags" in tw_item.keys():
         caldav_item["categories"] = tw_item["tags"]
 
+    # Project
+    if "project" in tw_item.keys():
+        # If there are tags use them to prevent overwritting
+        if "tags" in tw_item.keys():
+            project = tw_item["tags"]
+        else:
+            project = []
+        project.append("proj:" + tw_item["project"])
```

Comment:
could you add a space after `proj:` for readability (and also use `split(...).strip()` when reading it back afterwards)

Reply:
Do you mean split by spaces? Just for curiosity why we need that?